### DISCUSSION TITLE
How to avoid asking confirmation to press enter or quit when we run the code

### DISCUSSION THREAD

**Discussion by NajiAboo** [pycaret/pycaret#3064] [Q&A] [answered]
I am getting following message and I have to type enter to proceed. How to avoid this so that I can run smoothly.

Text(value="Following data types have been inferred automatically, if they are correct press enter to continue or type 'quit' otherwise.", layout=Layout(width='100%'))

**Comment by NajiAboo** [comment]
This happens when I run 
s = setup(data, target = 'charges', fold=2)
Then it's waiting to get confirmation from the user.  " Please Enter to confirm your request or 'Escape' to cancel

**Comment by NajiAboo** [comment, marked as answer]
I found the answer for this
s = setup(data, target = 'charges', fold=2, silent=True)

We need to add silent = True

**Comment by AZFARHAD24511** [comment]
I didnt find the answer for this
demo = setup(data = data, target = 'SalePrice', 
                   ignore_features = ['Alley','PoolQC','MiscFeature','Fence','FireplaceQu','Utilities'],normalize = True,
                   transformation= True, transformation_method = 'yeo-johnson', 
                   transform_target = True, remove_outliers= True,
                   remove_multicollinearity = True,
                   ignore_low_variance = True, combine_rare_levels = True,**silent=True**)
please tell me why?
